From 11bc907c3d336824d653690f6e686aa722bce3f7 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:04:31 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 68e42f6258f63684559addfb107f884ec82a4d09
Merge: f687799 a8e07ae
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jul 18 03:08:53 2018 +0000

    Snap for 4899218 from a8e07aecb24898d7d2b49cb785b0c193a4b134b4 to pi-dr1-release

    Change-Id: I9fb4cffa83625509c622653fea700ab93b46bc7f

commit a8e07aecb24898d7d2b49cb785b0c193a4b134b4
Author: Roger Liao <rogerliao@google.com>
Date:   Wed Jul 4 12:27:34 2018 +0800

    Android.mk: add sdm710

    Fixes the following build error:
    error: update_engine_sideload (EXECUTABLES android-arm64) missing
    bootctrl.sdm710 (STATIC_LIBRARIES android-arm64)

    Bug: 111163725
    Change-Id: I82f2321d580cb2fdb15d2343e39abed5ccda50b1

Change-Id: I3d3f7f710df8c9590776ebb686c83535716a7b9c
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 49efdae..51648f4 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,7 +1,7 @@
 ifeq ($(AB_OTA_UPDATER),true)
 # TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
 ifneq ($(BOARD_IS_AUTOMOTIVE),true)
-ifneq ($(filter msm8953 msm8996 msm8998 sdm660 sdm845,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8953 msm8996 msm8998 sdm660 sdm845 sdm710,$(TARGET_BOARD_PLATFORM)),)
 LOCAL_PATH := $(call my-dir)
 
 # HAL Shared library for the target. Used by libhardware.
-- 
2.17.1

